src/sonarcloud.ts

```diff
@@ -25,14 +25,26 @@ export const checkForErrors = (response: any) => {
   }
 };
 
-export const handleErrors = (errors: Error[]) => {
-  for (const error of errors) {
-    if (error.message.includes("No organization for key")) {
+type SonarcloudError = {
```

Comment:
Feels bad to need both of these, but I get the pragmatism.

Reply:
Yeah I would rather make it clear that this is sonarcloud's weird thing and that we cover both cases